structured.js

```diff
@@ -505,12 +506,53 @@
             }
             // Recursively check for matches
             if ((_.isArray(currTree[key]) &&
-                    checkNodeArray(currTree[key], toFind, peersToFind, wVars, matchResults, options)) ||
+                    checkNodeArray(currTree[key], toFind, peersToFind, wVars, matchResults, options, false)) ||
```

Comment:
Hm, does this one need false passed to it? It doesn't look like you modified checkNodeArray.